<?xml version="1.0"?>
<launch>

  <arg name="gripper_config_path"  default="/home/fshs/work/hirop/config/GripperConfig.yaml"/>

  <node pkg="gripper_bridge" type="gripper_bridge" name="gripper_bridge" output="screen">
     <param name="gripper_config_path" type="str" value="$(arg gripper_config_path)"/>
  </node>	

</launch>
